crates/ursa-pod/Cargo.toml

```diff
@@ -8,6 +8,8 @@ edition = "2021"
 [dependencies]
 rand.workspace = true
 anyhow.workspace = true
+arrayvec = "0.7.2"
+arrayref = "0.3.7"
```

Comment:
don't fix the minor version, these are crates developed by people who understand semver, not libp2p stack lol